src/window/basic_frame.rs

```diff
@@ -390,9 +390,24 @@ impl Frame for BasicFrame {
             // draw the grey background
             {
                 let mut writer = BufWriter::new(&mut *pool);
-                for _ in 0..pxcount {
-                    let _ = writer.write_u32::<NativeEndian>(color);
+                // For every pixel in top border
+                for y in 0..DECORATION_TOP_SIZE {
+                    for _ in 0..DECORATION_SIZE {
+                        writer.write_u32::<NativeEndian>(0x00_00_00_00);
```

Comment:
These writes generate compilation warnings, could you silence them with `let _ = ...` please?

Reply:
both have been fixed in the cleanup commit